### ISSUE TITLE
MySQL 8 new user is created without password if it doesn't meet requirements

### ISSUE THREAD

**Issue by adbario** [webmin/webmin#1126]
Latest MySQL 8 community release uses password validation by default (at least on CentOs 7).

When creating a new user and the password doesn't meet requirements, user will be created with an empty password.

Webmin will give an error, but I think the user sholudn't be created at all in this case:
```
Failed to save user : SQL set password for 'user'@'%' = 'password' failed : Your password does not satisfy the current policy requirements
```

Haven't tested this yet, but I believe on Virtualmin side there might be issues as well when passwords are generated automatically.

**Comment by jcameron** [comment]
Yes, I'd recommend lowering the requirements - otherwise Virtualmin will fail the user creation at some point.

**Comment by chris001** [comment]
If virtualmin's password generator were to include at least one punctuation mark, it should pass MySQL 8's out-of-the-box password requirements.

**Comment by iliaross** [comment]
❗️ If the issue isn't resolved, feel free to reopen it with updated details.
